enhancements/rhcos/rhcos-cloud-marketplace-images.md

```diff
@@ -0,0 +1,289 @@
+---
+title: marketplace-images
+authors:
+  - "@patrickdillon"
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - TBD
```

Comment:
Approved, would like to have lgtm from the following folks, or a delegate of their choosing, before we merge.
@jlebon or @mike-nguyen from RHCOS
@bennerv from ARO
@yuqi-zhang from MCO
@bryan-cox from HCP

Reply:
lgtm, but doublechecking with some other HyperShift folks to see if they are good with this as well.